aiohttp/__init__.py

```diff
@@ -203,11 +210,24 @@
     "TraceRequestRedirectParams",
     "TraceRequestStartParams",
     "TraceResponseChunkReceivedParams",
+    # workers (imported lazily with __getattr__)
+    "GunicornUVLoopWebWorker",
+    "GunicornWebWorker",
 )
 
-try:
-    from .worker import GunicornUVLoopWebWorker, GunicornWebWorker
 
-    __all__ += ("GunicornWebWorker", "GunicornUVLoopWebWorker")
```

Comment:
I guess `__dir__` should be overloaded also to keep the backward compatibility.

Reply:
How exactly should we do that? I notice here, they just add __doc__ and __author__ to __all__: https://github.com/python/cpython/blob/main/Lib/concurrent/futures/__init__.py

But, that's a lot less than what it currently returns. It currently includes a bunch of things we are not exporting (e.g. modules), plus several more things like __builtins__ and __path__.

Reply:
@webknjaz Any opinions on what we return from `__dir__`? Just do the same as the linked code above?

Reply:
Yeah, I guess that'd do.